src/main/java/org/apache/sling/feature/cpconverter/ContentPackage2FeatureModelConverter.java

```diff
@@ -286,11 +295,9 @@ public void convert(@NotNull File... contentPackages) throws IOException, Conver
             // analyze sub-content packages in order to filter out
             // possible outdated conflicting packages
             recollectorVaultPackageScanner.traverse(pack);
-
             logger.info("content-package '{}' successfully read!", contentPackage);
 
             aclManager.reset();
-            bundleSlingInitialContentExtractor.reset();
```

Comment:
Basically it didn't do anything in the past ... it was added because acl manager etc did reset something.. the paths were cleared but empty to begin with :(